keanu-project/src/test/java/io/improbable/keanu/algorithms/mcmc/nuts/NUTSTest.java

```diff
@@ -98,13 +94,12 @@ public void samplesGaussian() {
     @Test
     public void samplesContinuousPrior() {
```

Comment:
It's not *immediately* obvious what this test is actually verifying (or why what it's verifying is correct).  A bit of exposition might be nice to explain why we think the test is valid?

Reply:
For the gaussian and the sum of gaussian tests, it's asserting that the average sample is equal to the MAP. This is true because we know the posterior distribution is symmetric about the MAP. We know it's symmetric because we can analytically determine it due to it just being the same as the Prior (no observations), which is either a multivariate gaussian with a diagonal covariance matrix or just a simple gaussian. I'll drop this explanation in the test.

Reply:
Yep - agreed.  Thanks!